xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,7 +36,7 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
+  CPVRChannelGroup(bRadio, -1, g_localizeStrings.Get(19287)),
```

Comment:
I've already replaced magic numbers in a ton of places and they all have to go eventually. Do not introduce any new code with magic numbers like this please.